### ISSUE TITLE
Can't log out

### ISSUE THREAD

**Issue by NoooneyDude** [unitystation/unitystation#8868]
You can't log out from within the game.

![logout attempt](https://i.imgur.com/uXKu3aU.png)

```
MissingReferenceException: The object of type 'GameObject' has been destroyed but you are still trying to access it.
Your script should either check if it is null or you should not destroy the object.
UI.HighlanderTimerUI.Hide () (at Assets/Scripts/UI/Systems/MainHUD/HighlanderTimerUI.cs:44)
EventManager.Broadcast (Event evnt, System.Boolean network) (at Assets/Scripts/Managers/EventManager.cs:105)
UI.GUI_IngameMenu.LogoutConfirmYesButton () (at Assets/Scripts/UI/Systems/IngameMenu/GUI_IngameMenu.cs:201)
```

**Comment by Fr-Dae** [comment]
https://github.com/unitystation/unitystation/issues/8832

**Comment by NoooneyDude** [comment]
Closed by #8948.
